### PR TITLE
baseapp-core[graphql] docs

### PR DESCRIPTION
**Pull request by nossila**

### CODE REVIEW

File: baseapp-core/baseapp_core/graphql/README.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+### get_obj_relay_id
+
+Generate a relay id from a model instance, e.g.:
+
+```python
+from baseapp_core.graphql import get_obj_relay_id
+
+user = User.objects.get(pk=1)
+get_obj_relay_id(user)
```
Comment (anicioalexandre) [inline baseapp-core/baseapp_core/graphql/README.md:188]: if I inherit my model with `RelayModel` like [this](https://github.com/silverlogic/baseapp-backend/pull/54/files#diff-44f9d0f79db8c7b0d2f3812b00e16333009df89333462f2897ff61ae6f0c2bbfR132), we could get the same output if we just so a `user.relay_id`, right?

File: baseapp-core/baseapp_core/graphql/README.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+```
+    
+### get_obj_from_relay_id
+
+Get a model instance from a relay id, e.g.:
+
+```python
+from baseapp_core.graphql import get_obj_from_relay_id
+
+user = get_obj_from_relay_id("VXNlcjox")
```
Comment (anicioalexandre) [inline baseapp-core/baseapp_core/graphql/README.md:198]: doesn't this function require two parameters? `info` and `relay_id`

**Review by anicioalexandre — APPROVED** [review]
that's really helpful, great work!

File: baseapp-core/baseapp_core/graphql/README.md (hunk 1 above)
Comment (nossila) [reply baseapp-core/baseapp_core/graphql/README.md:188]: @anicioalexandre exactly!

File: baseapp-core/baseapp_core/graphql/README.md (hunk 2 above)
Comment (nossila) [reply baseapp-core/baseapp_core/graphql/README.md:198]: nice catch!

**Review by Janekk — APPROVED** [review]

**Review by pablogadhi — APPROVED** [review]

**Pull request merged**